Gemfile

```
@@ -2,66 +2,54 @@ source 'https://rubygems.org'
 
 ruby '2.6.6'
 
-# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '~> 5.0.0', '>= 5.0.0.1'
-# Use sqlite3 as the database for Active Record
+gem 'rails', '~> 5.2.0', '>= 5.0.0.1'
 gem 'pg', '~> 0.18'
-# Use Puma as the app server
-gem 'puma', '~> 3.12'
-# Use SCSS for stylesheets
-gem 'sass-rails', '~> 5.0'
-# Use Uglifier as compressor for JavaScript assets
-gem 'uglifier', '>= 1.3.0'
-# Use CoffeeScript for .coffee assets and views
-gem 'coffee-rails', '~> 4.2'
-# See https://github.com/rails/execjs#readme for more supported runtimes
-# gem 'therubyracer', platforms: :ruby
+gem 'puma'
 
-# Use jquery as the JavaScript library
-gem 'jquery-rails'
-# Turbolinks makes navigating your web application faster. Read more: https://github.com/turbolinks/turbolinks
-gem 'turbolinks', '~> 5'
-# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
-gem 'jbuilder', '~> 2.5'
-# Use Redis adapter to run Action Cable in production
-# gem 'redis', '~> 3.0'
-# Use ActiveModel has_secure_password
-# gem 'bcrypt', '~> 3.1.7'
+# Styles
+gem 'bootstrap-sass', '~> 3.4.1.0', '>= 3.3.7'
+gem 'sass-rails'
 
-# Use Capistrano for deployment
-# gem 'capistrano-rails', group: :development
+# Scripts
+gem 'coffee-rails'
+gem 'jquery-rails'
+gem 'turbolinks'
+gem 'uglifier'
 
-gem 'bootstrap-sass'
+# Monitoring
 gem 'newrelic_rpm'
 gem 'rollbar'
 
+# Api
 gem 'redis-rails'
+
+# Utilities
 gem 'rest-client'
 
 group :development, :test do
-  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
-  gem 'pry'
   gem 'dotenv-rails'
-  gem 'rspec-rails'
+  gem 'pry-byebug'
```

tento `pry-rails` prosim pridaj v novom PR aj do govboxu a harvestra

```suggestion
  gem 'pry-byebug'
  gem 'pry-rails'
```